src/main/java/io/xpire/model/Model.java

```diff
@@ -47,50 +51,58 @@
     /**
      * Returns the user prefs' xpire file path.
      */
-    Path getXpireFilePath();
+    Path getListFilePath();
 
     /**
      * Sets the user prefs' xpire file path.
      */
-    void setXpireFilePath(Path xpireFilePath);
+    void setListFilePath(Path xpireFilePath);
+
+    /**
+     * Returns an array containing xpire and replenish list.
+     */
+    ReadOnlyListView<? extends Item>[] getLists();
 
     /**
      * Replaces xpire data with the data in {@code xpire}.
      */
-    void setXpire(ReadOnlyXpire xpire);
+    void setXpire(ReadOnlyListView<XpireItem> xpire);
 
-    /** Returns the xpire */
-    ReadOnlyXpire getXpire();
 
     /**
-     * Returns true if an item with the same identity as {@code item} exists in xpire.
+     * Returns an Xpire object.
      */
-    boolean hasItem(Item item);
+    ReadOnlyListView<XpireItem> getXpire();
 
     /**
-     * Deletes the given item.
-     * The item must exist in xpire.
+     * Returns true if an xpireItem with the same identity as {@code xpireItem} exists in xpire.
      */
-    void deleteItem(Item target);
+    boolean hasItem(XpireItem xpireItem);
 
     /**
-     * Adds the given item.
-     * {@code item} must not already exist in xpire.
+     * Deletes the given xpireItem.
+     * The xpireItem must exist in xpire.
```

Comment:
Perhaps `The xpireItem` here can be replaced with {@code xpireItem} to match with the rest of the code :)

Reply:
Alright!